### PR TITLE
Feature/296 general design modifications

### PR DESCRIPTION
**Pull request by aouame**
Some aspects of the design changed  :

Text for tours is aligned left:
image

Fixed stats scrolling ( the same goes for tours nearby and gallery )

image

image

Changed some font and size of text (quizz options , audio, mapfragment text) , align vertically the quizz options indexes :

image

Added a padding in the bottom of start tour cardview:

image

Added some space between experiences: 
image

Ar title and subtitle got centered too.

### CODE REVIEW

**Comment by qtipee** [comment]
On the home activity: is it possible to reduce the shadow on the experience fragments ? Or increase the margin between two items ? We can clearly see the frontier between two items and it is not pretty. Moreover, I do not think the « tours near me » right margin has the same behaviour than the Stats Block.

About the Quiz Block: the letters are not vertically aligned.

**Comment by aouame** [comment]
> On the home activity: is it possible to reduce the shadow on the experience fragments ? Or increase the margin between two items ? We can clearly see the frontier between two items and it is not pretty. Moreover, I do not think the « tours near me » right margin has the same behaviour than the Stats Block.
> 
> About the Quiz Block: the letters are not vertically aligned.

Yes I'll tackle these too.

File: app/src/main/AndroidManifest.xml [hunk 1]
```diff
@@ -21,7 +21,7 @@
         android:name=".MainApplication"
         android:allowBackup="true"
         android:icon="@mipmap/pindex_launcher"
-        android:label="@string/app_name"
+        android:label=""
```
Comment (hadiyahya11) [inline app/src/main/AndroidManifest.xml:24]: Just delete this attribute ?

**Review by hadiyahya11 — DISMISSED** [review]
Nice Pr, the code is to the point. The design is better now. Just delete not needed lines

**Review by aouame — COMMENTED** [review]
One point remaining is the title "Pindex" on every activity. I did not manage to get it off without deleting the app's name, so I kept it.

**Review by lucasrlt — DISMISSED** [review]
Everything looks really nice, well done ! Especially after the correction of the feedbacks we noticed while recording the video

**Review by hadiyahya11 — APPROVED** [review]

**Review by ToufiPF — APPROVED** [review]
LGTM

**Pull request merged**